sra-schema/test_df.csv

```diff
@@ -0,0 +1,28 @@
+,Agent that provides to QBIC,Existing property in OpenBIS,New property in OpenBIS,OpenBIS object type,Metadata Level,Specific metadata,SRA Structured comment name,SRA Item,EGA metadata object,EGA item,EGA label,Definition,Expected value,OpenBIS type,json type,Value syntax,Example,Section,migs_eu,migs_ba,migs_pl,migs_vi,migs_org,me,mimarks_s,mimarks_c,misag,mimag,miuvig,ega,Plant,Virus,SarsCov,PathogenEnv,PathogenCl,Microbe,Metagenome,Invertebrate,Human,BetaLactamase,Unnamed: 40,Unnamed: 41,Unnamed: 42,Unnamed: 43,Unnamed: 44,Unnamed: 45,Unnamed: 46,Unnamed: 47,Unnamed: 48
```

Comment:
Could the row numbers be removed from this table? Is that a test table? Or what does `test_df` mean?